cluster/addons/registry/registry-pv.yaml.in

```diff
@@ -0,0 +1,16 @@
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+  name: kube-system-kube-registry-pv
+  labels:
+    kubernetes.io/cluster-service: "true"
+spec:
+{% if pillar.get('cluster_registry_disk_type', '') == 'gce' %}
```

Comment:
There's a pattern to do this e.g. http://stackoverflow.com/questions/21778252/how-to-raise-an-exception-in-a-jinja2-macro, but I'm not sure we have any of the pieces elsewhere in our Salt. This looks fine for now, though, it'll error in a different way, I think.